pkg/models/model.go

```diff
@@ -142,8 +143,12 @@ func (m *Model) downloadSpagoModel() error {
 	if err != nil {
 		return err
 	}
+
 	if !modelsPathExists {
-		return fmt.Errorf("models path %#v does not exist", modelsPath)
+		err = os.Mkdir(modelsPath, 0755)
+		if err != nil {
+			return fmt.Errorf("can't create folder %v %s", modelsPath, err)
```

Comment:
Just because I feel obnoxious :stuck_out_tongue_winking_eye: (and also for pedantic consistency), here is a subtle variation:

```suggestion
			return fmt.Errorf("can't create folder %#v: %w", modelsPath, err)
```

Motivation:
- `%#v` is just for beauty, so that the path is enclosed in double quotes `"..."`, which is nice if it includes whitespaces;
- wrapping the error with `%w` is currently irrelevant, but it might be useful for the caller to unwrap the error and do something useful with it (in the future, or by some project using this as a library).